Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:14.04
+MAINTAINER Elico Corp <info@elico-corp.com>
+
+# generate locales
+RUN locale-gen en_US.UTF-8 && update-locale
+RUN echo 'LANG="en_US.UTF-8"' > /etc/default/locale
+
+# Add the PostgreSQL PGP key to verify their Debian packages.
+# It should be the same key as https://www.postgresql.org/media/keys/ACCC4CF8.asc
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
+
+# Add PostgreSQL's repository. It contains the most recent stable release
+#     of PostgreSQL, ``9.4``.
+# install dependencies as distrib packages when system bindings are required
+# some of them extend the basic odoo requirements for a better "apps" compatibility
+# most dependencies are distributed as wheel packages at the next step
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+  apt-get update && \
+  apt-get -yq install \
+    adduser \
+    ghostscript \
+    postgresql-client-9.4 \
+    python \
+    python-pip \
+    python-imaging \
+    python-pychart python-libxslt1 xfonts-base xfonts-75dpi \
+    libxrender1 libxext6 fontconfig \
+    python-zsi \
+    python-lasso \
+    # SM: libpq-dev is needed to install pg_config which is required by psycopg2
+    libpq-dev \
+    # SM: These libraries are needed to install the pip modules
+    python-dev \
+    libffi-dev \
+    libxml2-dev \
+    libxslt1-dev \
+    libldap2-dev \
+    libsasl2-dev \
+    libssl-dev \
+    # SM: This library is necessary to upgrade PIL/pillow module
+    libjpeg8-dev \
+    # SM: Git is required to clone Odoo OCB project
+    git
+
+ADD sources/pip-req.txt /opt/sources/pip-req.txt
+
+# SM: Install Odoo python dependencies
+RUN pip install -r /opt/sources/pip-req.txt
+
+# SM: Upgrade pillow to allow JPEG resize operations (used by demo data)
+RUN pip install -i --upgrade pillow
+
+# must unzip this package to make it visible as an odoo external dependency
+RUN easy_install -UZ py3o.template
```

Comment:
I didn't change what was working. I only updated what wasn't. This is 
working.
Sébastien Maillard --
*Sébastien Maillard* /[Technical Analyst]/
Phone: + 86 186 2103 7589 (Cell), + 86 21 6211 8017/27/37 (Office)
*Elico Shanghai* (Shenzhen/Singapore) <http://www.elico-corp.com> /*Odoo 
Gold Partner, best Odoo Partner 2014 for APAC*/
On 09/25/2015 05:38 PM, Augustin Cisterne-Kaas wrote:
>
> In Dockerfile 
> <https://github.com/Elico-Corp/odoo-docker/pull/4#discussion_r40414068>:
>
> > +    libssl-dev \
> > +    # SM: This library is necessary to upgrade PIL/pillow module
> > +    libjpeg8-dev \
> > +    # SM: Git is required to clone Odoo OCB project
> > +    git
> > +
> > +ADD sources/pip-req.txt /opt/sources/pip-req.txt
> > +
> > +# SM: Install Odoo python dependencies
> > +RUN pip install -r /opt/sources/pip-req.txt
> > +
> > +# SM: Upgrade pillow to allow JPEG resize operations (used by demo data)
> > +RUN pip install -i --upgrade pillow
> > +
> > +# must unzip this package to make it visible as an odoo external dependency
> > +RUN easy_install -UZ py3o.template
>
> Can't you do it with |pip| instead of |easy_install| ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Elico-Corp/odoo-docker/pull/4/files#r40414068>.
>